src/main.cpp

```diff
@@ -3,24 +3,29 @@
 #include <stdio.h>
 #include <SDL.h>
 
-//how to use:
-//./<binary> xpos ypos
+
+namespace cameraVariables
+{
+    SDL_Window* window_ = NULL;
```

Comment:
Instead of using a global variable, let's pass this into BackupCamera as a member variable.